src/CL/Slack/Resources/config/serializer/CL.Slack.Payload.ImOpenPayloadResponse.yml

```diff
@@ -4,3 +4,5 @@ CL\Slack\Payload\ImOpenPayloadResponse:
             type: boolean
         alreadyOpen:
             type: boolean
+        channel:
+            type: array
```

Comment:
No, I believe on the slack im.open method returns a response with a channel and not a imchannel. 

https://api.slack.com/methods/im.open